### DISCUSSION TITLE
How to smoothly close a `Swipeable`?

### DISCUSSION THREAD

**Discussion by Myzel394** [software-mansion/react-native-gesture-handler#2255] [Q&A]
https://user-images.githubusercontent.com/50424412/193633901-5c05f1c0-c13c-4b37-b7e3-0aaee858b51c.mp4

Take a look at this video - When I swipe completely to the left or the right, the `close` method on the `Swipeable` is called. The problem is, it closes instantly. How can I create a nice (bouncy) close animation?

**Comment by willashley23** [comment]
Bump

**Comment by m-bert** [comment]
Hi! We've recently reimplemented `Swipeable` using `react-native-reanimated`. Could you check if it behaves better?
